.circleci/config.yml

```diff
@@ -815,8 +815,7 @@ jobs:
           name: Publish a dev release
           command: |
             source .venv/bin/activate
-            bin/release-helper.sh set-ver $(bin/release-helper.sh next-dev-ver)
-            make publish
+            make dev-publish || echo "Not publishing a dev release"
```

Comment:
I think we don't need an additional make target here, but can really just run `make publish` but either check if the version is already published, or ignore failures (because if it fails, it's basically always because the version already exists).